### PR TITLE
[Enhancement]: Update dockerfile and add github release note

### PR DESCRIPTION
**Pull request by RunningLeon**
## Motivation

Update dockerfile and add github release note

## Modification

- Update prebuild dockerfile
- Add github release note yml

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by lvhan028** [comment]
any idea how to resolve `codecov/patch`?

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+changelog:
```
Comment (lvhan028) [inline .github/release.yml:1]: when will `release.yml` be invoked?

**Comment by irexyc** [comment]
shoud update `image: openmmlab/mmdeploy:build-ubuntu18.04-cuda11.3` to `image: openmmlab/mmdeploy:build-ubuntu16.04-cuda11.3` in prebuild.yml

File: .github/release.yml (hunk 1 above)
Comment (RunningLeon) [reply .github/release.yml:1]: on github release page, there is a button for triggering it manually.

**Review by irexyc — APPROVED** [review]

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**